lib/routes/token.js

```diff
@@ -46,12 +46,18 @@ const ID_TOKEN_KEY = JwTool.JWK.fromObject(config.get('openid.key'), {
 
 const REFRESH_LAST_USED_AT_UPDATE_AFTER_MS = config.get('refreshToken.updateAfter');
 
+const BASIC_AUTH_REGEX = /^Basic\s+([A-Za-z0-9+\/]+)$/i;
```

Comment:
Minor nit, but this regex sets the ignore-case flag then goes to the effort of including both cases in the character class. Presumably one or other of those is unnecessary?

Reply:
Indeed, nice catch!  I used `/i` for matching the "Basic" part but didn't consider the rest.